rskj-core/src/main/java/co/rsk/peg/federation/FederationSupportImpl.java

```diff
@@ -587,6 +601,14 @@ private Integer addFederatorPublicKeyMultikey(boolean dryRun, BtcECKey btcKey, E
         return FederationChangeResponseCode.SUCCESSFUL.getCode();
     }
 
+    protected FederationChangeResponseCode commitFederationAccordingToActivations(boolean dryRun, Keccak256 hash, BridgeEventLogger eventLogger) {
```

Comment:
I'd consider renaming the `hash` param. Sounds too generic

Reply:
done